<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportDICOMPanelWidget</class>
 <widget class="QWidget" name="ImportDICOMPanelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="importDICOMInstructionsTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>301</width>
     <height>131</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Import DICOM&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Select an input directory that contains the DICOM objects. That input directory and its subdirectories will be searched for DICOM objects that can be converted to images.&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Select an output directory for storing the converted images. A hierarchy of subdirectories will be automatically created that correspond to the &amp;lt;PatientName&amp;gt;/&amp;lt;StudyID&amp;gt;-&amp;lt;StudyDescription&amp;gt;-&amp;lt;StudyDate&amp;gt;/&amp;lt;Modality&amp;gt;/&amp;lt;SeriesNumber&amp;gt;-&amp;lt;SeriesDescription&amp;gt;-&amp;lt;InstanceNumber&amp;gt;.dcm&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Optionally, the data can be automatically registered with Minder3D and made available for one-click loading via the image browser table.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="importDICOMInputDirectoryLineEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="importDICOMInputDirectoryLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input Directory:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="importDICOMInputDirectorySelectButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>20</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="importDICOMOutputDirectorySelectButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>50</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="importDICOMOutputDirectoryLineEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="importDICOMOutputDirectoryLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output Directory:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="importDICOMAutoRegisterCheckBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Automatically register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="importDICOMRunButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>100</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
